src/video/wayland/SDL_waylandwindow.c

```diff
@@ -43,12 +43,63 @@ static float get_window_scale_factor(SDL_Window *window) {
       return ((SDL_WindowData*)window->driverdata)->scale_factor;
 }
 
+static void
+CommitMinMaxDimensions(SDL_Window *window)
+{
+    SDL_WindowData *wind = window->driverdata;
+    SDL_VideoData *data = wind->waylandData;
+    int min_width, min_height, max_width, max_height;
+
+    /* FIXME: Is there a better way to get max window size from Wayland? -flibit */
+    const int maxsize = 0x7FFFFFFF;
```

Comment:
If you want to tell the Wayland compositor/server not to constrain the maximum size, just send 0: `xdg_toplevel_set_max_size(..., 0, 0)`. There is no need to figure out the maximum size.